bin/clean_output

```diff
@@ -0,0 +1 @@
+rm -rf ./*/config*.dot ./*/*.dot.png
```

Comment:
Is this supposed to remove files with the extensions  ".dot" and ".png"? I used it in a directory with the files config.Mouse.txt.dot.png,  expected.dot,  and expected.dot.png but they all remained in the directory.

Reply:
It should not remove the expected.dot file. The idea behind this script is to use it from the root of the PipeViz directory.